sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -645,8 +645,21 @@ class DataSourceV2SQLSuiteV1Filter
       assert(replaced.columns.length === 1,
         "Replaced table should have new schema.")
       val actual = replaced.columns.head
-      val expected = ColumnV2.create("id", LongType, false, null,
-        new ColumnDefaultValue("41 + 1", LiteralValue(42L, LongType)), null)
+      val expected = ColumnV2.create(
```

Comment:
After this change `InMemoryTable` exposes actual Columns, so we actually see the passed "current" expressions too.